src/main/java/com/urbanairship/datacube/dbharnesses/HBaseDbHarness.java

```diff
@@ -400,7 +410,14 @@ private void flushBatch(Batch<T> batch) throws IOException, InterruptedException
             }
         }
 
+        long startTime = System.currentTimeMillis();
         final Result[] results = WithHTable.get(pool, tableName, gets);
+        long cubeReadLatency = System.currentTimeMillis() - startTime;
+
+        multiGetIdServicesLatency.update(idServicesLatency, TimeUnit.MILLISECONDS);
+        multiGetCubeLatency.update(cubeReadLatency, TimeUnit.MILLISECONDS);
+        multiGetIdServicesLatencyRatio.update(idServicesLatency / (cubeReadLatency + idServicesLatency));
```

Comment:
Given the quirks of integer division, I'd probably make this a percentage. something like `(long) (idServicesLatency / ((double) cubeReadLatency + idServicesLatency) * 100)`